BaseDataContactsCSV.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace test1
+{
+
+    public class BaseDataContactsCSV : IDataContactInterface
+    {
+        private string _nameFile = string.Empty;
+        private int _countLine = 0;
+        private readonly Regex _separatorChar = new("[^;]+", RegexOptions.Compiled);
+        
+        private bool _flagTryAmout = false;
+        private int _amoutOfContact = 0;
+
+        public bool TryInitializationDB(string? nameFile)
+        {
+            if (File.Exists($"{nameFile}.csv"))
+            {
+                _nameFile = $"{nameFile}.csv";
+                _countLine = AmountOfContact();
+                return true;
+            }
+            return ValidationImputClass.TryValidatoinNameFile(nameFile, out _nameFile);
```

Comment:
```suggestion
            return ValidationInputClass.TryValidatoinNameFile(nameFile, out _nameFile);
```
От слова In - put, т.е. по сути "засунуть"

Reply:
поправил